Update docs for Android August release

dshokouhi opened the pull request:
Please only merge this once the play store release is live!

- Removes all beta tags for the android release
- Updates all notification commands to the new format along with keeping the old parameter table and mentioning that things changed in 2022.8

jpelgrom commented:
On the notification commands page, you've added the sentence "If you have updated the android app to 2022.8 you must use `command` in place of `title` like the below example:" a few times, but no example follows (usually it's the title of the next command). 

Also seeing "android" a few times, try using an uppercase A 🙂

dshokouhi commented:
> On the notification commands page, you've added the sentence "If you have updated the android app to 2022.8 you must use `command` in place of `title` like the below example:" a few times, but no example follows (usually it's the title of the next command).

Thats right I added the examples then I realized I was just duplicating the example directly above it so I removed them but forgot to remove the mention of "the example" lol



> Also seeing "android" a few times, try using an uppercase A 🙂

oops ill get those fixed

jpelgrom commented:
2022.8.0 is now available from the Play Store

jpelgrom commented on docs/notifications/commands.md:
```diff
@@ -264,9 +226,9 @@ automation:
           data:
-            channel: "package-name"
+            intent_package_name: "package-name"
+            intent_action: "action
```
Missing a `"` at the end
